Sources/Views/PricingLanding.swift

```diff
@@ -134,81 +150,88 @@ private let contactusButtonClasses =
     | Class.border.all
     | Class.pf.colors.border.gray800
 
-private let plansAndPricing = [
```

Comment:
lots of churn in this file due to re-indenting some stuff. you may want to hide white space changes in this PR review